internal/services/compute/linux_virtual_machine_resource.go

```diff
@@ -662,6 +663,10 @@ func resourceLinuxVirtualMachineRead(d *pluginsdk.ResourceData, meta interface{}
 			if err := d.Set("admin_ssh_key", pluginsdk.NewSet(SSHKeySchemaHash, *flattenedSSHKeys)); err != nil {
 				return fmt.Errorf("setting `admin_ssh_key`: %+v", err)
 			}
+			patchSettings := config.PatchSettings
+			if patchSettings != nil && patchSettings.PatchMode != "" {
+				d.Set("patch_mode", patchSettings.PatchMode)
+			}
```

Comment:
we need to ensure this field always has a value set into the state, so this'll need to become:

```suggestion
			patchMode := string(compute.LinuxVMGuestPatchModeImageDefault)
			if patchSettings := config.PatchSettings; patchSettings != nil && patchSettings.PatchMode != "" {
				patchMode = *patchSettings.PatchMode
			}
			d.Set("patch_mode", patchMode)
```